.github/workflows/python-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]

```

Comment:
Make sure we use updated versions based on latest stable version or as per standards if defined.

Reply:
This can be ignored, as it's a sample example. User can set the latest version or the version used in their project at the time of implementation.